contracts/Vault.vy

```diff
@@ -437,7 +449,6 @@ def setDepositLimit(limit: uint256):
     self.depositLimit = limit
     log UpdateDepositLimit(limit)
 
-
```

Comment:
keep the double line spacing

Reply:
fixed 05d5a59c07b731d8148f287f978b640bbd87dff8